live/tests/acceptance/not-found-redirect-to-index-test.js

```diff
@@ -0,0 +1,26 @@
+import { afterEach, beforeEach, describe, it } from 'mocha';
+import { startApp, destroyApp } from '../helpers/application';
+import { expect } from 'chai';
+
+describe('Acceptance | Page | Not Found Redirection', () => {
+
+  let application;
+
+  beforeEach(() => {
+    application = startApp();
+  });
+
+  afterEach(() => {
+    destroyApp(application);
+  });
+
+  it('unexistant page should redirect to home page', () => {
```

Comment:
Un `it` commence toujours par "should".
Renommer en "should redirect to home page when entered URL does not exist"